legal-api/src/legal_api/resources/v2/business/business_filings/business_filings.py

```diff
@@ -265,6 +265,37 @@ def patch_filings(identifier, filing_id=None):
     return jsonify(filing.json), HTTPStatus.ACCEPTED
 
 
+@bp.route('/filings/search', methods=['POST'])
+@cross_origin(origin='*')
+@jwt.has_one_of_roles([UserRoles.staff])
+def get_fed_filing():
+    """Return a single FED filing."""
```

Comment:
I think this endpoint isn't unique to FE filings, so I'd generalize the name and comment.